source/precalculus/source/06-TR/04.ptx

```diff
@@ -12,6 +12,632 @@
+          <answer>
+            <p>
+              D
+            </p>
+          </answer>
+        </task>
+
+        <task>
+          <statement>
+            <p>
+              In triangle <m>ABC</m>, <m>B=90</m>,<degree/>, <m>b=6\sqrt{2}</m>, and <m>C=45</m><degree/>. What is the length of <m>c</m>?
+              <ol marker= "A." cols="2">
+                <li> <m>6</m> </li>
+                <li> <m>5</m></li>
+                <li> <m>4</m> </li>
+                <li> <m>12</m> </li></ol>
+            </p>
+          </statement>
+          <answer>
+            <p>
+              A
+            </p>
+          </answer>
+        </task>
+
+        <task>
+          <statement>
+            <p>
+              In triangle <m>ABC</m>, <m>B=90</m>,<degree/>, <m>b=4</m>, and <m>A=45</m><degree/>. What is the length of <m>c</m>?
+              <ol marker= "A." cols="2">
+                <li> <m>\sqrt{6}</m> </li>
+                <li> <m>2\sqrt{2}</m></li>
+                <li> <m>3</m> </li>
+                <li> <m>6</m> </li></ol>
+            </p>
+          </statement>
+          <answer>
+            <p>
+              B
+            </p>
+          </answer>
+        </task>
+      </activity>
+
+      <activity xml:id="exploring-30-60-90-triangle">
+        <introduction>
+          <p>
+            Suppose you are given a <m>30-60-90</m> triangle, where you know the angle opposite the <m>30</m><degree/> angle is of length <m>1</m>.
+          </p>
+        </introduction>
+        <task>
+          <statement>
+            <p>
+              Draw a right triangle and label the angles to have <m>30</m><degree/>, <m>60</m> <degree/>, and <m>90</m><degree/> and the side opposite of the <m>30</m><degree/> angle as having a length of <m>1</m>.
+            </p>
+          </statement>
+          <answer>
+            <p>
+              Students should be able to draw a right triangle with one <m>90</m><degree/> angle (indicated by a box), one <m>30</m><degree/> angle and one <m>60</m><degree/> angle. The side opposite the <m>30</m><degree/> angle should have a side length of <m>1</m>.
+            </p>
+          </answer>
+        </task>
+        <task>
+          <statement>
+            <p>
+              Unlike the <m>45-45-90</m> triangle, we cannot easily find another side length with the information given. Using trigonometry, which trigonometric function would help us find the length of the hypotenuse with the information given?
+              <ol marker= "A." cols="1">
+                <li> <m>\sin\theta</m> </li>
+                <li> <m>\cos\theta</m></li>
+                <li> <m>\tan\theta</m> </li></ol>
+            </p>
+          </statement>
+          <answer>
+            <p>
+              A
+            </p>
+          </answer>
+        </task>
+
+        <task>
+          <statement>
+            <p>
+              If we use the trigonometric ratio from part (b), what is the length of the hypotenuse? 
```

Comment:
I don't think I understand the intent here, are you just wanting them to put this in their calculator?

Reply:
I was trying to incorporate in some way that these radicals they see are lengths of the side of the triangle. I thought maybe if they plugged it into a calculator, they could see that there is an approximation (and a size) that they have a hard time seeing when it is in radical form. Should I maybe just add something at the end that asks for an approximation of the size of the length of the triangle?